dissect/target/plugins/os/unix/esxi/_os.py

```diff
@@ -239,17 +239,22 @@ def _decrypt_envelope(local_tgz_ve: TargetPath, encryption_info: TargetPath) ->
     return local_tgz
 
 
-def _decrypt_crypto_util(local_tgz_ve: TargetPath) -> BytesIO:
-    """Decrypt ``local.tgz.ve`` using esxi ``crypto-util``."""
-
-    local_tgz = BytesIO(
-        subprocess.run(
-            ["crypto-util", "envelope", "extract", "--aad", "ESXConfiguration", f"/{local_tgz_ve.as_posix()}", "-"],
-            capture_output=True,
-        ).stdout
+def _decrypt_crypto_util(local_tgz_ve: TargetPath) -> Optional[BytesIO]:
+    """Decrypt ``local.tgz.ve`` using ESXi ``crypto-util``.
+
+    Using subprocess run will return a non-zero return_code, where stderr contains an an I/O error message.
+    However, the file gets properly decrypted, so we return ``None`` if there are no bytes in stdout.
```

Comment:
```suggestion
    We write to stdout, but this results in ``crypto-util`` exiting with a non-zero return code and stderr containing an  I/O error message.
    The file does get properly decrypted, so we return ``None`` if there are no bytes in stdout which would indicate it actually failed.
```
Or something like that? My best guess on what's happening.